[2025-02-28T22:17:31.804+0200] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-28T22:17:31.809+0200] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_token_processing.consume_tokens scheduled__2025-02-28T20:10:00+00:00 [queued]>
[2025-02-28T22:17:31.812+0200] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_token_processing.consume_tokens scheduled__2025-02-28T20:10:00+00:00 [queued]>
[2025-02-28T22:17:31.812+0200] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-28T22:17:31.816+0200] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): consume_tokens> on 2025-02-28 20:10:00+00:00
[2025-02-28T22:17:31.823+0200] {logging_mixin.py:190} WARNING - /Users/nampham/.pyenv/versions/3.12.9/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=16193) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-28T22:17:31.822+0200] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_token_processing', 'consume_tokens', 'scheduled__2025-02-28T20:10:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/kafka_token_processing.py', '--cfg-path', '/var/folders/dw/z1bbk7h952586r5vt211lpjc0000gn/T/tmp7fagsnmr']
[2025-02-28T22:17:31.823+0200] {standard_task_runner.py:72} INFO - Started process 16199 to run task
[2025-02-28T22:17:31.823+0200] {standard_task_runner.py:105} INFO - Job 9: Subtask consume_tokens
[2025-02-28T22:17:31.837+0200] {task_command.py:467} INFO - Running <TaskInstance: kafka_token_processing.consume_tokens scheduled__2025-02-28T20:10:00+00:00 [running]> on host nams-macbook-air.local
[2025-02-28T22:17:31.858+0200] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_token_processing' AIRFLOW_CTX_TASK_ID='consume_tokens' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T20:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T20:10:00+00:00'
[2025-02-28T22:17:31.858+0200] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-28T22:17:31.858+0200] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-28T22:17:31.859+0200] {logging_mixin.py:190} INFO - Current task name:consume_tokens state:running start_date:2025-02-28 20:17:31.810095+00:00
[2025-02-28T22:17:31.859+0200] {logging_mixin.py:190} INFO - Dag name:kafka_token_processing and current dag run status:running
[2025-02-28T22:17:31.859+0200] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-28T22:17:31.859+0200] {kafka_token_processing.py:37} INFO - Running consumer task...
[2025-02-28T22:17:41.973+0200] {kafka_token_processing.py:41} INFO - Consumer task output: Waiting for message...
Waiting for message...
Waiting for message...
Waiting for message...
Waiting for message...
Waiting for message...
Waiting for message...
Waiting for message...
Waiting for message...
Waiting for message...

[2025-02-28T22:17:41.975+0200] {python.py:240} INFO - Done. Returned value was: None
[2025-02-28T22:17:41.984+0200] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-28T22:17:41.985+0200] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=kafka_token_processing, task_id=consume_tokens, run_id=scheduled__2025-02-28T20:10:00+00:00, execution_date=20250228T201000, start_date=20250228T201731, end_date=20250228T201741
[2025-02-28T22:17:41.995+0200] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-28T22:17:41.996+0200] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-28T22:17:41.996+0200] {logging_mixin.py:190} INFO - Dag name:kafka_token_processing queued_at:2025-02-28 20:15:02.507541+00:00
[2025-02-28T22:17:41.996+0200] {logging_mixin.py:190} INFO - Task hostname:nams-macbook-air.local operator:PythonOperator
[2025-02-28T22:17:42.043+0200] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-28T22:17:42.056+0200] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-28T22:17:42.057+0200] {local_task_job_runner.py:245} INFO - ::endgroup::
